Task/Workflow/TSKWorkflow.h

```diff
@@ -142,13 +144,13 @@ extern NSString *const TSKWorkflowTaskKey;
 @property (nonatomic, strong, readonly) NSNotificationCenter *notificationCenter;
```

Comment:
notificationCenter is never null because if you specify nil, we use the default center. If you provide a nil name, we pick a default for you.